### PR TITLE
add region ap-southeast-3

### PR DESCRIPTION
**Pull request by ilhamderi**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by iwysiu — COMMENTED** [review]
Hi @ilhamderi ! Thanks for contributing. Can you create an issue for this on the project?

File: pkg/tsdb/cloudwatch/metrics.go [hunk 1]
```diff
@@ -510,7 +510,7 @@ var dimensionsMap = map[string][]string{
 // Known AWS regions.
 var knownRegions = []string{
 	"af-south-1", "ap-east-1", "ap-northeast-1", "ap-northeast-2", "ap-northeast-3", "ap-south-1", "ap-southeast-1",
-	"ap-southeast-2", "ca-central-1", "cn-north-1", "cn-northwest-1", "eu-central-1", "eu-north-1", "eu-south-1", "eu-west-1",
+	"ap-southeast-3", "ap-southeast-2", "ca-central-1", "cn-north-1", "cn-northwest-1", "eu-central-1", "eu-north-1", "eu-south-1", "eu-west-1",
```
[the remark this answers is not in the document]
Comment (ilhamderi) [reply pkg/tsdb/cloudwatch/metrics.go]: I need to get metric from cloudwatch in region ap-southeast-3 (new region in december 2021), but in options default regions not yet available

**Comment by andresmgot** [comment]
Can you please sign the CLA @ilhamderi ?

**Comment by ilhamderi** [comment]
> Can you please sign the CLA @ilhamderi ?

oh sure

**Comment by andresmgot** [comment]
not sure if the CLA check got stuck, did you sign it @ilhamderi ?

**Comment by daniellee** [comment]
@ilhamderi is the email address you use in Git different than the one connected to your GitHub account?

image

**Comment by yaelleC** [comment]
Closing as change made in: https://github.com/grafana/grafana/pull/45821

**Pull request closed without merging**